main.go

```diff
@@ -175,6 +175,13 @@ func main() {
 		os.Exit(1)
 	}
 
+	if cn, err := getClusterName(); err == nil {
+		if cl, err := getClusterLocality(); err == nil {
+			nodeMetadata["GKE_CLUSTER_NAME"] = cn
```

Comment:
Isn't this already available in deploymentInfo's `GKE-CLUSTER`? (I don't claim to understand the difference between zone and cluster-location.) Are we just not using that because we'll need to add cluster location?

Reply:
yes, and `TRAFFIC_DIRECTOR_CLIENT_ENVIRONMENT` is currently optional field added to node metadata,  controlled by a flag. We would want this to be added by default when run on GKE.

Reply:
Well, it could be enabled by default. Do we know what is going on with it and why we are doing something different here?

Reply:
The person who owns the design is OOO until next week. The discussion is blocked until then

Reply:
Finalized the discussion in the doc. Using `TRAFFIC_DIRECTOR_CLIENT_ENVIRONMENT` metadata to add cluster info.